DOC-852: New iframe_aria_text setting (#2033)
tinymce/tinymce-docs: advanced/accessibility.md

### Navigating dialogs

Dialogs such as **Insert/Edit Image** are opened from either a menu item or a toolbar button. Some dialogs contain multiple tabs or pages of options. To change tabs or pages, use the arrow keys; the highlighted tab will become active immediately. Use the **Tab** key to navigate between the dialog options on the selected dialog tab. The **Save** dialog button stores the changes, and the **Cancel** dialog button or the **Esc** key discards changes.

## Accessibility options

> {{site.productname}} has accessibility options that can be set when initialising the editor, see: [Accessibilty options]({{site.baseurl}}/configure/accessibility/).

Reviewer: ```suggestion
{{site.productname}} has accessibility options that can be set when initializing the editor, see: [Accessibility options]({{site.baseurl}}/configure/accessibility/).
```

### Navigating dialogs

Dialogs such as **Insert/Edit Image** are opened from either a menu item or a toolbar button. Some dialogs contain multiple tabs or pages of options. To change tabs or pages, use the arrow keys; the highlighted tab will become active immediately. Use the **Tab** key to navigate between the dialog options on the selected dialog tab. The **Save** dialog button stores the changes, and the **Cancel** dialog button or the **Esc** key discards changes.

> 

Reviewer 2: ```suggestion
```

## Accessibility options

> TinyMCE has accessibility options that can be set when initialising the editor, see: [Accessibilty options]({{site.baseurl}}/configure/accessibility-options/).

Reviewer 2: ```suggestion
{{site.productname}} has accessibility options that can be set when initialising the editor, see: [Accessibilty options]({{site.baseurl}}/configure/accessibility-options/).
```
We should use the `site.productname` variable instead of `TinyMCE` in the content.
